packages/blockchain-wallet-v4-frontend/src/data/components/utils/selectors.ts

```diff
@@ -30,7 +30,8 @@ export const getSupportedCoinsWithBalanceAndOrder = (state: RootState) => {
       mapObjIndexed((coin: SupportedWalletCurrencyType) => {
         return {
           ...coin,
-          method: coin.coinCode in CoinTypeEnum || balances[coin.coinCode]
+          method: coin.coinCode in CoinTypeEnum || balances[coin.coinCode],
+          availableBalance: balances[coin.coinCode]?.available !== '0'
```

Comment:
Im making a small change to use only `method`, because that will be true if they've ever held a balance. But availableBalance could become false if their balance goes to 0